test/db/tools/rz_diff

```diff
@@ -200,38 +200,38 @@ sym.imp.__ctype_get_mb_cur_max    6 0x0000000000401150 | MATCH   (1.000000) | 0x
               sym.imp.iswprint    6 0x0000000000401370 | MATCH   (1.000000) | 0x0000000000401370     6 sym.imp.iswprint
          sym.imp.__ctype_b_loc    6 0x0000000000401380 | MATCH   (1.000000) | 0x0000000000401380     6 sym.imp.__ctype_b_loc
                   fcn.00401460   41 0x0000000000401460 | MATCH   (1.000000) | 0x0000000000401470    41 fcn.00401470
-                  fcn.00401530  745 0x0000000000401530 | UNMATCH (0.198093) | 0x0000000000401540   745 fcn.00401540
-                  fcn.004018f0  147 0x00000000004018f0 | UNMATCH (0.122449) | 0x0000000000401900   147 fcn.00401900
-                  fcn.00401990  137 0x0000000000401990 | UNMATCH (0.233577) | 0x00000000004019a0   137 fcn.004019a0
+                  fcn.00401530  745 0x0000000000401530 | SIMILAR (0.879195) | 0x0000000000401540   745 fcn.00401540
```

Comment:
So essentially it as giving precisely the opposite of the valid result, right?

Reply:
yes

Reply:
before the double value was overwritten by the next compare. 
Specifically this line: https://github.com/rizinorg/rizin/pull/1346/files#diff-320081f996f71252b81a941f8e29742888f632f5da0ae5114a89cfd64d5a2b01L248

Reply:
🤦